README.md

```diff
@@ -714,6 +714,25 @@ We have explicitly observed the following:
   However, in some cases we use caching and in other cases it was not always possible
   to group everything for reasons of clarity.
 
+### Build Capella from source
+
+We provide a script, which runs a Docker container to build Capella from source.
+Our builder script adds the aarch64 build platform for macOS and linux.
+Please note that Capella itself has no official support for the aarch64 architecture yet.
+We can not guarantee that everything works, but all of the official Capella tests passed on the generated archives.
+
+In addition, the builder only works for Capella version 6.0.0 for now.
+To start the script, please run:
+
+```
+CAPELLA_VERSION=6.0.0 make capella/builder
+```
+
+When the script is successful, you should see the result in `builder/output`.
+
+We store a cache of the maven repository in the `builder/m2_cache` directory.
+If you don't need the cache anymore, you can delete the directory after building.
+
```

Comment:
How can I limit the build to one or a set of platforms?
I just ran a build and got

```text
capella-6.0.0.202302172253-linux-gtk-aarch64.tar.gz
capella-6.0.0.202302172253-linux-gtk-x86_64.tar.gz
capella-6.0.0.202302172253-macosx-cocoa-aarch64.tar.gz
capella-6.0.0.202302172253-macosx-cocoa-x86_64.tar.gz
capella-6.0.0.202302172253-win32-win32-x86_64.zip  
```
That took 35 minutes and I was just interested in one build.

Reply:
There is no way to limit it - and there is also not a significant speed improvement when building against one platform. There are just a few small platform specific dependencies which have to be pulled for each platform individually - all additional dependencies are platform independent. 

Therefore, I decided that it doesn't really make sense to just build against one platform - if you don't need the other platforms, just delete the files.

Reply:
Ah, okay, thanks! Hoped that one could save a lot of build time.